### PR TITLE
Fix for #190 - created artwork for Principles page

### PR DESCRIPTION
**Pull request by sbddesign**

### CODE REVIEW

**Comment by sbddesign** [comment]
There's some alternate header designs in the Figma file if anyone is interested in seeing:
[Figma File](https://www.figma.com/file/xQ5eWVSOLbNGLmcls3qz7d/Principles?node-id=0%3A1)

**Comment by pavlenex** [comment]
Visually, they look great! 😍 

Do you think we should re-size them for example like we did here https://deploy-preview-453--sad-borg-390916.netlify.app/guide/designing-products/usage-life-cycle/#unaware ?

File: Gemfile.lock [hunk 1]
```diff
@@ -118,7 +118,7 @@ GEM
 PLATFORMS
   ruby
   x86_64-linux
-  x86_64-linux-musl
```
Comment (johnsBeharry) [inline Gemfile.lock:121]: Was it a mistake to commit this Gemfile change? If it was I'd make a new commit with whats on master instead so we don't end up with such changes that may go untested.

**Review by johnsBeharry — COMMENTED** [review]
@sbddesign awesome work man, love the style of these icons so much.

File: Gemfile.lock (hunk 1 above)
Comment (sbddesign) [reply Gemfile.lock:121]: Accident, I'll fix.

**Comment by sbddesign** [comment]
@pavlenex I made the images inline. It looked a little weird having the images inline with the paragraph text, so I customized it a little bit and made them sit next to the blockquotes.

**Review by Bosch-0 — CHANGES_REQUESTED** [review]
LGTM, though I think the icons look better centered to the block quote. Currently they float slightly above.

image

**Comment by pavlenex** [comment]
Posting this comment from Christoph 

> I don't think the formatting with the block quotes next to the images work. How about using bold text instead of block quotes?

Which may solve the alignment problem we're having.

**Comment by sbddesign** [comment]
The above commit is an attempt to address feedback in this thread as well as in the discussion thread for #190.

**Review by pavlenex — APPROVED** [review]
tACK

**Pull request merged**